keras_hub/src/models/segformer/segformer_image_segmenter_preprocessor.py

```diff
@@ -0,0 +1,31 @@
+import keras
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.image_segmenter_preprocessor import (
+    ImageSegmenterPreprocessor,
+)
+from keras_hub.src.models.segformer.segformer_backbone import SegFormerBackbone
+from keras_hub.src.models.segformer.segformer_image_converter import (
+    SegFormerImageConverter,
+)
+from keras_hub.src.utils.tensor_utils import preprocessing_function
+
+IMAGENET_DEFAULT_MEAN = [0.485, 0.456, 0.406]
+IMAGENET_DEFAULT_STD = [0.229, 0.224, 0.225]
+
+
+@keras_hub_export("keras_hub.models.SegFormerImageSegmenterPreprocessor")
+class SegFormerImageSegmenterPreprocessor(ImageSegmenterPreprocessor):
+    backbone_cls = SegFormerBackbone
+    image_converter_cls = SegFormerImageConverter
+
+    @preprocessing_function
+    def call(self, x, y=None, sample_weight=None):
+        if self.image_converter:
+            x = self.image_converter(x)
+            y = self.image_converter(y)
```

Comment:
wouldn't this also rescale the masks if rescale is part of image_converter?

Reply:
Yes, it would. The ImageConverter should only resize - rescaling and normalization is enabled in the preprocessor by default, since it's required for the weights to function as intended. This should at the very least be documented, or ideally set as the default to avoid confusion.

If I add a default ImageConverter with just resizing and export the SegFormer model as a task again - the ImageConverter's `from_preset()` would restore it to that state as well? In that case, it may make the most sense to export the SegFormer models with the ImageConverter configs already baked in. Thoughts?